pdr_backend/lake/test/test_table.py

```diff
@@ -1,241 +1,149 @@
-from io import StringIO
 import os
-import sys
-from polars import Boolean, Float64, Int64, Utf8
-import polars as pl
+
 from pdr_backend.ppss.ppss import mock_ppss
 from pdr_backend.lake.table import Table
-from pdr_backend.subgraph.subgraph_predictions import fetch_filtered_predictions
-from pdr_backend.lake.table_pdr_predictions import predictions_schema
-from pdr_backend.util.time_types import UnixTimeMs
-
-
-# pylint: disable=too-many-instance-attributes
-class MyClass:
-    def __init__(self, data):
-        self.ID = data["ID"]
-        self.pair = data["pair"]
-        self.timeframe = data["timeframe"]
-        self.prediction = data["prediction"]
-        self.payout = data["payout"]
-        self.timestamp = data["timestamp"]
-        self.slot = data["slot"]
-        self.user = data["user"]
-
-
-mocked_object = {
-    "ID": "0x123",
-    "pair": "ADA-USDT",
-    "timeframe": "5m",
-    "prediction": True,
-    "payout": 28.2,
-    "timestamp": 1701634400000,
-    "slot": 1701634400000,
-    "user": "0x123",
-}
-
-
-def mock_fetch_function(
-    network, st_ut, fin_ut, save_backoff_limit, pagination_limit, config
-):
-    print(network, st_ut, fin_ut, save_backoff_limit, pagination_limit, config)
-    return [MyClass(mocked_object)]
+from pdr_backend.lake.table_pdr_predictions import (
+    predictions_schema,
+    predictions_table_name,
+)
+from pdr_backend.lake.persistent_data_store import PersistentDataStore
+from pdr_backend.lake.csv_data_store import CSVDataStore
+
 
+def _table_exists(persistent_data_store, table_name):
+    table_names = persistent_data_store.get_table_names()
+    return [table_name in table_names, table_name]
 
-def get_table_df(network, st_ut, fin_ut, config):
-    print(network, st_ut, fin_ut, config)
-    return pl.DataFrame([mocked_object], table_df_schema)
 
+def _clean_up_persistent_data_store(tmpdir, table_name):
```

Comment:
this seems like a violation of DRY with the other function doing the exact same thing in lake/test_resources